src/mpid/ch4/src/ch4_comm.h

```diff
@@ -201,6 +203,10 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_Comm_create_hook(MPIR_Comm * comm)
         }
     }
 
+#if defined HAVE_LIBHCOLL
+    hcoll_comm_create(comm, NULL);
+#endif
+
```

Comment:
It'll require some more changes in order to move it to ucx layer - for now we're just trying to validate the PR